Python/password-manager.pyw

```diff
@@ -0,0 +1,151 @@
+            # appends character to list randomly
+            new_list.append(random.choice(
+                '0123456789!@#$%^&*_abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ'))
+
+        random.shuffle(new_list)  # shuffles whole list toincrease randomness
+
+        password = ""
+
+        password = password.join(new_list)  # converts list to a string
+
+        def copy():  # function to copy password to clipboard
+
+            pyperclip.copy(password)  # copies password
+
+            Button(root, text="Copied", height=1, width=10, relief="raised", bd=8,
+                   bg="white", command=copy, fg="BLUE", state="disabled").grid(row=11, column=1)
+
+        def save():  # function to save password in new file
+
+            Button(root, text="Save", height=1, width=10, relief="raised", bd=8,
+                   command=save, bg="white", fg="BLUE", state="disabled").grid(row=11, column=2)
+
+            Label(root, text="    Password is for platform : ",
+                  font=("Slab Serif", 16)).grid(row=16, column=0)
+
+            c = StringVar()  # take domain as input form user
+
+            Entry(root, textvariable=c).grid(row=16, column=1)
+
+            def confirm():
+
+                copy()  # function calling
+
+                password = pyperclip.paste()
+
+                Button(root, text="Saved", height=1, relief="raised", width=11, bd=8, bg="white",
+                       command=confirm, state="disabled", fg="BLUE").grid(row=16, column=2)
+
+                data = [c.get(), password]
+
+                file = open('passwords.csv', 'a')
+
+                csv.writer(file).writerow(data)
+
+                file.close()
+
+            Button(root, text=">>", height=1, width=11, relief="raised", bg="white", bd=8,
+                   activebackground="blue", command=confirm, fg="BLUE").grid(row=16, column=2)
+
+        Label(root, text="      Your random Password is : ",
+              font=("Slab Serif", 16)).grid(row=5, column=0)
+
+        Button(root, text="Copy", relief="raised", height=1, width=10, bd=8, bg="white",
+               activebackground="blue", command=copy, fg="BLUE").grid(row=11, column=1)
+
+        Button(root, text="Save & Copy", relief="raised", height=1, width=11, bd=8,
+               bg="white", activebackground="blue", command=save, fg="BLUE").grid(row=11, column=2)
+
+        return new_list
+
+
+def res():
+
+    Label(root, text=passwordgenerate(), font=("Slab Serif", 16),).grid(
+        row=5, column=1, columnspan=2, padx=20)
+
+    Button(root, text="Generate", height=3, width=18, bd=8, activebackground="blue", command=res,
+           relief="raised", bg="white", state="disabled", fg="blue").grid(row=30, column=60,)
+
+
+button = Button(root, text="Generate", height=3, width=18, bd=8, activebackground="blue",
+                command=res, relief="raised", bg="white", fg="blue").grid(row=30, column=60,)
+
+
+root.mainloop()  # loop that allows GUI to run
```

Comment:
- The code is fine, do the changes suggested in the beginning. Also, attach a GIF showing the working of the program, its functions as right now it's difficult to judge whether it is working or not. Put both the files (GIF and code file in a folder and put that folder under the Python directory. 
- Update the language folder readme too :-)